docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
+3. Responses MUST have the same value for the producedAt and thisUpdate times.
+4. The difference in time between the thisUpdate and nextUpdate fields MUST be greater than or equal to 8 hours and MUST NOT be greater than or equal to 192 hours (7 days).
+5. For OCSP responses directly signed by the issuing CA, the response MUST NOT include any certificates within the BasicOCSPResponse's certs field.
+6. For OCSP responses signed by a Delegated OCSP Responder Certificate, the response MUST only include the Delegated OCSP Responder Certificate within the BasicOCSPResponse's certs field and MUST NOT include any additional certificates.
```

Comment:
This may require some rethinking about how CAs produce OCSP responses for non-TLS certificates. For non-TLS certificates, it may be useful to include the full chain in the OCSP response for OCSP archival. However, that's totally not useful for TLS certificates, and actively harmful to performance.